app/addons/documents/mango/components/ExplainPage.js

```diff
@@ -23,10 +26,175 @@ export default class ExplainPage extends Component {
     prettyPrint();
   }
 
+  state = {
+    viewFormat: 'parsed',
+    isReasonsModalVisible: false,
```

Comment:
`viewFormat` must be part of the redux state (not local state). Ideally both should be but I'm ok with the modal one being local since it's ok for it to reset if the page reloads.